server/src/main/scala/org/labrad/browser/RegistryApi.scala

```diff
@@ -22,7 +22,19 @@ object RegistryApi {
   }
 }
 
+/**
+ * The contents of a registry directory, potentially including recursive subdirectory contents
+ */
+case class RegistryContents(dirs: Seq[(String, RegistryContents)], keys: Seq[(String, Data)])
+object RegistryContents {
+  val Empty = RegistryContents(dirs = Nil, keys = Nil)
+}
```

Comment:
@maffoo I see the pattern over and over but I don't get it. What's the point of making a case class and an object of the same name?